src/style.css

```diff
@@ -1,3 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+:root {
+  --hoverColor: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
```

Comment:
Check whether you deployed changes or not, since by demo link I found a difference 
<img width="1259" alt="Screenshot 2023-06-22 at 21 40 48" src="https://github.com/mate-academy/layout_moyo-header/assets/117587463/93822abf-1720-4430-a4e5-1d7b6d3bdc1e">

Reply:
У мене чомусь не виходить поновити ці лінки до поточного стану проекту. Як це зробити?